### ISSUE TITLE
IDP Broken: Unable to configure Max Clock Skew for okta_idp_saml resource

### ISSUE THREAD

**Issue by Venkatsai** [okta/terraform-provider-okta#194]
The Identity Provider when created on okta tenant is broken due to the fact the Max Clock Skew value is set to null through terraform. There should be an argument which can define under okta_idp_saml resource.

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

Terraform v0.12.29
provider.okta v3.5.0

### Affected Resource(s)

* okta_idp_saml

### Expected Behavior

The okta_idp_saml should have argument for Max Clock Skew and user should have flexibility choosing the value.

```
resource "okta_idp_saml" "test" {
  username_template  = "idpuser.subjectNameId"
  subject_match_type = "USERNAME"
  profile_master     = true
  groups_action      = "SYNC"
  groups_attribute   = "group_name"
  groups_filter      = ["group_A", "group_B"]
  kid                = okta_idp_saml_key.example.kid
  name               = "IDP"
  acs_binding        = "HTTP-POST"
  sso_url            = "https://hub.okta.com/app/okta_org2org/xxxxx/sso/saml"
  issuer             = "http://www.okta.com/xxxx"
 max_clock_skew = 1 minutes or milliseconds or seconds
}
```

### Actual Behavior

The max clock skew attribute on the okta tenant is set to null

### Steps to Reproduce

```
resource "okta_idp_saml" "test" {
  username_template  = "idpuser.subjectNameId"
  subject_match_type = "USERNAME"
  profile_master     = true
  groups_action      = "SYNC"
  groups_attribute   = "group_name"
  groups_filter      = ["group_A", "group_B"]
  kid                = okta_idp_saml_key.example.kid
  name               = "IDP"
  acs_binding        = "HTTP-POST"
  sso_url            = "https://hub.okta.com/app/okta_org2org/xxxxx/sso/saml"
  issuer             = "http://www.okta.com/xxxx"
}
```

1. `terraform apply`

**Comment by ghost** [comment]
Just faced this issue as well, this is breaking SAML2 IDPs, any updates?
![image](https://user-images.githubusercontent.com/21358910/99648354-a004ed00-2a5b-11eb-9978-71719c91af4e.png)

**Comment by bogdanprodan-okta** [comment]
Hi @Venkatsai, @IrmantasM! I will look into this issue shortly. It's likely to be fixed in v3.7.4
